README_new.md

```diff
@@ -0,0 +1,132 @@
+
+## 1. babylon demo project
+
+From a high level point of view the demo consists of a list of posts, where each post has its own detail.
+
+#### Posts Screen
+
+A post has a title and it's up to you how to display it. To retrieve the posts, you can use the following:
+
+* http://jsonplaceholder.typicode.com/posts
+
+When a post is tapped, you should go to the detail screen.
+
+#### Detail screen
+
+A post detail screen, will have:
+
+* Its author.
+* Its description (via the `body`).
+* Number of comments it has.
+
+You can retrieve the remaining information from:
+
+* http://jsonplaceholder.typicode.com/users
+* http://jsonplaceholder.typicode.com/comments
+
+#### Requirements
+
+The following requirements should be met:
+
+* Use Swift 3.0.
+* The information (posts and post details) should be available offline. It's assumed that if it's the first time you are accessing the app, and you are offline, you shouldn't see anything.
+* The code should be production grade.
+* It should compile and run.
+
+## 2. Already written project
+
+We would be happy if you would submit a project you already have (e.g. potentially a demo project for another company). Still  the project has to obey to the following requirements:
+
+* Use Swift 3.0.
+* It should have at least two distinct network calls.
+* It should parse the network response and present the information to the user.
+* It should have some sort of persistence mechanism.
+* It should compile and run.
+* **Ideally** it should have a point of synchronization (e.g. making two concurrent requests and waiting for both of them to finish).
+
+If you have a project with these requirements then perfect! Please ensure you have a description of what the project does, in order to give us some context. ❤️
+
+## 3. Open Source work
+
+We would like to see a **non-trivial** pull request you have done to a public open source project. This should be something you are proud and where you show your technical skills. **It should also be related to iOS development**: it should be aligned with what you will do on a day-to-day basis. 😊✨🌳
+
+---
+
+## General Tips/Advice
+
+* We like code that is simple, [but simple is different than easy](https://www.infoq.com/presentations/Simple-Made-Easy).
+* Keep in mind the [SOLID principles](https://en.wikipedia.org/wiki/SOLID_(object-oriented_design)) when doing the project.
+* We did leave out of the requirements if you should try to download everything (posts + each post detail). This is up to you to decide and justify why you did so.
+* Testing is very important for us. Even if you don't write a single test (e.g. because of time constraints), your app should be easy to test (we love [Dependency injection](https://en.wikipedia.org/wiki/Dependency_inversion_principle)).
+* Error scenarios should be taken into consideration as well and how easy is to add them, even if you don't explicitly show them (e.g. using an `UIAlertController`).
```

Comment:
Do we love `Dependency Inversion` or `Dependency Injection` or both?

Reply:
Well we stated that we appreciate `SOLID` principles :)

Reply:
I suppose both, since they're different concepts. In my defence, I didn't write any of the text starting with `babylon demo project` :) (that's from the original demo project page)

Reply:
Both